packages/networks/src/networks.ts

```diff
@@ -63,6 +63,63 @@ export const networks = {
     protocolMultisig: undefined,
   },
   // TEST NETWORKS
+  mantleTestnetSepolia: {
+    name: 'Mantle Testnet Sepolia',
+    type: NetworkType.testnet,
+    key: 'mantleTestnetSepolia',
+    shortKey: 'mantleTestnet',
+    color: '#B026FF',
+    chain: 5003,
+    rpc: process.env.MANTLE_TESTNET_SEPOLIA_RPC_URL || 'https://rpc.sepolia.mantle.xyz',
+    holographId: 4000000079,
```

Comment:
This holographId is the same as polygonTestnetAmoy currently. Please make them unique between the 2.

Reply:
good catch

Reply:
updated